Makefile

```diff
@@ -18,4 +18,10 @@ check:
 
 all: functional
 
+checkcommits:
+	cd pkg/checkcommits/ && go build && go test
```

Comment:
Not sure about putting `checkcommits` in `pkg/`, that's usually reserved for non main packages. Those are put in a `cmd` directory.